doc/getting_started.md

```diff
@@ -0,0 +1,109 @@
+# Getting started
+
+<!-- TOC -->
+
+- [Getting started](#getting-started)
+    - [SPIRE artifacts](#spire-artifacts)
+    - [Building the artifacts](#building-the-artifacts)
+    - [Installing your artifact in Ubuntu 16.04](#installing-your-artifact-in-ubuntu-1604)
+    - [Configuration](#configuration)
+    - [Running SPIRE with join token](#running-spire-with-join-token)
+
+<!-- /TOC -->
+
+## SPIRE artifacts
```

Comment:
Before getting to this, presumably one needs to have a basic idea of the SPIRE architecture (server/agent) and how attestation works. Could we include something here to that effect?

Reply:
One way to do this is to set up the tutorial context eg. "To provide a minimal example of how SPIRE can be used, we are going to set up a single-node spire server and use it to issue identities to a workload identified by being run under a specified user lD, and on a manually identified machine.

Reply:
Also worth identifying expected pre-requisites (eg. we expect you to have two VMs or physical machines running Ubuntu 16.02 that can connect to each other)

Reply:
Thanks for all the comments, much appreciated! 
Only one question. Should we facilitate a Vagrantfile? It wouldn't be hard to create it since I've been using one for testing all this.

Reply:
I think this doc should be as environment-agnostic as possible, so no Docker, Vagrant etc

Reply:
Agree with @drrt - lets keep this as vanilla as possible.